platform/mds/mds-web/src/main/resources/webapp/messages/messages.properties

```diff
@@ -224,8 +224,10 @@ mds.regex.dateTimeInfo=DateTime formats: yyyy[/.-]mm[/.-]dd hh:mm:ss
 mds.delete.deleteEntityTitle=Delete Entity
 mds.delete.deleteEntity=Selected entity and all its metadata will be deleted
 mds.delete.deleteInstanceTitle=Delete Instance
+mds.delete.deleteSelectedInstancesTitle=Delete Selected Instances
 mds.delete.deleteBlobContentTitle=Delete BLOB content
 mds.delete.deleteInstance=Selected instance will be deleted. Proceed?
+mds.delete.deleteSelectedInstances=All the selected instances will be deleted. Proceed?
```

Comment:
Can we add the number of selected instances into this string? Its confusing to read this message...